engines/cpu.c

```diff
@@ -63,14 +180,57 @@ static enum fio_q_status fio_cpuio_queue(struct thread_data *td,
 		return FIO_Q_BUSY;
 	}
 
-	usec_spin(co->cpucycle);
+	switch (co->cpumode) {
```

Comment:
I do prefer having separate lines, makes it easier to read for a switch.

Reply:
done